### PR TITLE
Feature/ni3 go back btn

### LINKED ISSUE #1: [2️⃣ ] When a movie’s details are displayed, the user should have a way to return to the main view of all movies

### PR DESCRIPTION
**Pull request by natesheridan**
## [Rancid Tomatillos 🍅]

- [ ] - Feature PR
- [X] - Refactor PR
- [ ] - Other PR

**What (if any) features are you implementing?**

 - (NA)
 
**What (if anything) did you refactor?**

 - Button to leave singleMovieView functionally
 - Closes #1 

**Were there any issues that arose?** 

 - (NA)

**Is there anything needs to be prioritized?**

 - (NA)

**Is there anything that you need from your teammates?**

 - (NA)

**GOALS?**

 - (NA)

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -27,13 +27,16 @@ class Main extends Component {
     const selectedMovie = this.state.movies.find(movie => movie.id === id);
     this.setState({ selectedMovie: selectedMovie });
   }
+  goBack = () => {
+    this.setState({ selectedMovie: ''})
+  }
```
Comment (elisebeall) [inline src/components/Main.js:33]: I think this is a pretty unique way to go about returning to all of the movies.  I think that the lesson on router will really make this part of React even cleaner.

File: src/components/SingleMovieScreen.js [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
 import '../css/SingleMovieScreen.css';
 
-const SingleMovieScreen = (movie) => {
-  console.log(movie)
+const SingleMovieScreen = (props) => {
   return(
     <article className="single-movie-view"
       style={{
-        backgroundImage: `url(${movie.movie.backdrop_path})`
+        backgroundImage: `url(${props.movie.backdrop_path})`
```
Comment (elisebeall) [inline src/components/SingleMovieScreen.js:8]: Glad we were able to get rid of the "double movie" :)

File: src/css/SingleMovieScreen.css [hunk 3]
```diff
@@ -7,9 +7,47 @@
...
+	position:relative;
+	top:1px;
+}
+.single-movie-back-btn img{
+  position: relative;
+  height: 100%;
+  right: 20px;
+  width: auto;
+  z-index: 1;
+}
```
Comment (elisebeall) [inline src/css/SingleMovieScreen.css:47]: Woweee.  That's a stylish button right there.

**Comment by elisebeall** [comment]
@kaylagordon
@robbiejaeger

We have a very persistent bug where this.state.movies tries to populate the cards before the data returns from the api.  At least I believe that's what's happening.  I will message you both on Slack to share more details.

File: src/components/MovieListContainer.js [hunk 4]
```diff
@@ -12,10 +12,9 @@ class MovieListContainer extends Component {
...
 
   returnCardElements() {
-    //console.log(this.state.movies);
-    let movieCardElementArr = this.state.movies.map((movie, i) => {
-      let classes = `movie-card fade-in animation-duration:${i}00ms`;
-      return <MovieCard
+      let movieCardElementArr = this.state.movies.map((movie, i) => {
+        let classes = `movie-card fade-in animation-duration:${i}00ms`;
+        return (<MovieCard
         key={movie.id}
```
Comment (robbiejaeger) [inline src/components/MovieListContainer.js:18]: Great job using the `id` of the movie here as the key.

File: src/components/SingleMovieScreen.js [hunk 5]
```diff
@@ -1,31 +1,33 @@
...
       }}>
 
 
       <div className="cover-poster">
-        <img src={movie.movie.poster_path} alt={movie.movie.title} />
+        <img src={props.movie.poster_path} alt={props.movie.title} />
```
Comment (robbiejaeger) [inline src/components/SingleMovieScreen.js:13]: Consider destructuring props so that you can reduce the number of times you have to use `props.`

**Pull request merged**